deploy/examples/cluster.yaml

```diff
@@ -93,15 +93,15 @@ spec:
     # enable the Multus network provider
     #provider: multus
     #selectors:
-      # The selector keys are required to be `public` and `cluster`.
-      # Based on the configuration, the operator will do the following:
-      #   1. if only the `public` selector key is specified both public_network and cluster_network Ceph settings will listen on that interface
-      #   2. if both `public` and `cluster` selector keys are specified the first one will point to 'public_network' flag and the second one to 'cluster_network'
-      #
-      # In order to work, each selector value must match a NetworkAttachmentDefinition object in Multus
-      #
-      #public: public-conf --> NetworkAttachmentDefinition object name in Multus
-      #cluster: cluster-conf --> NetworkAttachmentDefinition object name in Multus
+    # The selector keys are required to be `public` and `cluster`.
+    # Based on the configuration, the operator will do the following:
+    #   1. if only the `public` selector key is specified both public_network and cluster_network Ceph settings will listen on that interface
+    #   2. if both `public` and `cluster` selector keys are specified the first one will point to 'public_network' flag and the second one to 'cluster_network'
+    #
+    # In order to work, each selector value must match a NetworkAttachmentDefinition object in Multus
+    #
+    #public: public-conf --> NetworkAttachmentDefinition object name in Multus
```

Comment:
Did your editor change all this whitespace? It's good to preserve some of the indentation for the comments so the user can see that the indentation is important for properties like this.